server/tests/database/test_organization_data_handler.py

```
@@ -0,0 +1,38 @@
+from flaskr.database import OrganizationDataHandler
+
+
+def test_create_organization(app):
```

Ruff (0.15.15)

[warning] Unused function argument: `app`

(ARG001)